web-server/src/components/AnimatedInputWrapper/AnimatedInputWrapper.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+
+import { themeColors } from '@/theme/schemes/theme';
+
+import styles from './slider.module.css';
+
+const AnimatedInputWrapper = () => {
+  return (
+    <div>
+      <AnimatedPlaceHolderWrapper />
+      <input
+        className={styles.placeholder}
+        style={{ backgroundColor: themeColors.primaryAlt }}
+      />
```

Comment:
Why are you rendering this input? it doesn't seem to show up anywhere.
I guess you need to add some width and height to the `<div>` component in `AnimatedInputWrapper` component, and remove `input` completely.

Feel free to ask more about this.

Reply:
used a div instead of input in the latest commit